frontend/src/components/doctors-filtration-panel/doctors-filtration-panel.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import { useForm } from 'react-hook-form';
+import styles from './doctors-filtration.module.scss';
+import location from 'assets/images/location.svg';
+import specialty from 'assets/images/specialty.svg';
+import clinic from 'assets/images/clinic.svg';
+import { TextInput, Checkbox } from 'components/common';
+import { DoctorType, ClinicType, InputType, InputColor, DoctorFiltration } from 'common/enums';
+import { IDoctorFiltrationPayload } from 'common/interfaces';
+import Details from 'components/common/details/details';
+
+const DEFAULT_VALUES: IDoctorFiltrationPayload = {
+  [DoctorFiltration.DOCTOR_NAME]: '',
+  [DoctorFiltration.CITY]: '',
+  [DoctorFiltration.SPECIALTY]: [],
+  [DoctorFiltration.TYPE_OF_CLINIC]: ClinicType.STATE
+}
```

Comment:
Move it to the `frontend/src/components/doctors-filtration-panel/common/constants`